### DISCUSSION TITLE
Batch Cut multiple Videos with pre defined times

### DISCUSSION THREAD

**Discussion by chickchecker** [mifi/lossless-cut#1228] [General]
Hello,

i have hunreds of Videos and have to cut them in one step. Is it possible to let losslesscut do the job.

For example:

i want to cut every 5 Minutes 10 seconds off every Video. I this possibe? I do not want to edit every Video, i just want an automatic cut of multiple Videos at pre defined timestamps and export them to new Videos.

Thank you

**Comment by mifi** [comment]
hi. I think you're better off using ffmpeg directly from the command line with bash:
```bash
for f in *.mp4; do ffmpeg -t 310 -i "$f" -c copy "$f-out.mp4"; done
```

**Reply by chickchecker** [reply]
does not work, shows 2 Errors
